### PR TITLE
Manage Tests page table view

### PR DESCRIPTION
**Pull request by jacbn**
Replaces the cards (one per test, per group) in the Manage Tests page (/set_tests) with a table view. Each row of the outer table represents one test, and on click expands to show each group to which that test has been assigned, and relevant options for that group.

### CODE REVIEW

**Review by mlt47 — APPROVED** [review]
This is functionally good and useful. I've made a few suggested tweaks for the styling.

File: src/app/components/pages/quizzes/SetQuizzes.tsx [hunk 1]
```diff
@@ -38,56 +40,188 @@ interface SetQuizzesPageProps extends RouteComponentProps {
...
+                    }}
+                >
+                    {siteSpecific("Set Test", "Set test")}
+                </RS.Button>
+            </td>
+            <td className={`dropdown-arrow ${isExpanded ? "active" : ""}`}/>
+        </tr>
+        {isExpanded && <tr>
+            <td colSpan={4} className={classNames("bg-white border-0", {"px-2 pb-2": isPhy})}>
+                <RS.Table striped className="w-100 set-quiz-table-inner">
```
Comment (mlt47) [inline src/app/components/pages/quizzes/SetQuizzes.tsx:185]:
```suggestion
                <RS.Table striped className="w-100 set-quiz-table-inner mb-1">
```

**Review by mlt47 — APPROVED** [review]
Great!

File: src/app/components/pages/quizzes/SetQuizzes.tsx [hunk 2]
```diff
@@ -39,56 +41,193 @@ interface SetQuizzesPageProps extends RouteComponentProps {
...
+                                </div>
+                            </th>)}
+                            <th className="px-1 py-1" colSpan={2}>Actions</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        {conditionalReverse(assignedGroups.sort(currentSort)).map(assignedGroup => {
+                        const assignmentNotYetStarted = assignedGroup.assignment?.scheduledStartDate && nthHourOf(0, assignedGroup.assignment?.scheduledStartDate) > TODAY();
+                        return <tr key={assignedGroup.group}>
+                            <td>{assignedGroup.group}</td>
```
Comment (mlt47) [inline src/app/components/pages/quizzes/SetQuizzes.tsx:210]: Groups with long names bleed into the next cell so I would recommend adding some CSS that includes `overflow-wrap: anywhere;` or similar.

**Review by mlt47 — APPROVED** [review]
Some minor suggestions, but otherwise looks good.

File: src/scss/phy/isaac.scss [hunk 3]
```diff
@@ -160,7 +160,7 @@ $question-font-weight: 600;
 @import "assignment-progress";
 @import "books";
 @import "menu-page-phy";
-@import "quiz";
```
[the remark this answers is not in the document]
Comment (jacbn) [reply src/scss/phy/isaac.scss:163]: Not sure why this changed, it should be using the filter styling in phy/quiz.scss.

**Pull request merged**